lte/gateway/python/magma/pipelined/app/vlan_egress_flow_msgs.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
Move this file up one directory and rename it to vlan_utils.py

Reply:
:+1: pushed